.github/workflows/cicd.yml

```
@@ -150,7 +150,7 @@ jobs:
 
 
   unit:
-    name: Unit Tests using Python ${{ matrix.python-version }} on Ubuntu with Twisted ${{ matrix.twisted }}
+    name: "Twisted ${{ matrix.twisted }} on Python ${{ matrix.python-version }} on ${{ matrix.os }}"
```

Maybe 
```suggestion
    name: "TX ${{ matrix.twisted }} - PY ${{ matrix.python-version }} - ${{ matrix.os }}"
```

As devs should know what each means :)